bokeh/tests/test_io.py

```diff
@@ -293,3 +295,13 @@ def _test_children_removed_from_root(layout_generator, children=None):
     layout_generator(component if children is None else children)
     assert component not in io.curdoc().roots
     io.curdoc().clear()
+
+def test__get_screenshot_as_png(monkeypatch):
+    def mockreturn(self):
+        return "PNG"
+    monkeypatch.setattr(selenium.webdriver.remote.webdriver.WebDriver, "get_screenshot_as_png", mockreturn)
+
+    layout = Plot(x_range=Range1d(), y_range=Range1d())
+
+    png = io._get_screenshot_as_png(layout)
+    assert png == "PNG"
```

Comment:
I'm unsold on this test. I think it might be better to actually create a tiny, empty Plot and compare ``png`` against an expected binary string. If the plot is empty (a white square), there shouldn't be any test failures due to changing defaults.